src/peft/tuners/trainable_tokens/config.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2025-present the HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import Optional, Union
+
+from peft.config import PeftConfig
+from peft.utils import PeftType
+
+
+@dataclass
+class TrainableTokensConfig(PeftConfig):
+    """
+    Configuration for the `TrainableTokens` method.
+
+    This method makes certain tokens (identified by their indices) trainable without as large a memory footprint as
+    training the target matrix fully. The following arguments are need.
```

Comment:
```suggestion
    training the target matrix fully.
```

I think the docstring should mention prominently that this is intended for training new tokens in the embedding matrix.